DOCUMENTATION.md

```diff
@@ -13,30 +13,32 @@ If you want to read more into `flatpak override`, you can look at the [`flatpak-
 
 ### Permissions
 
+This is a list of permissions configurable in Flatseal. The descriptions below will assume that the option is enabled (toggled on), if it's a toggle.
+
 #### Share
 
 List of subsystems shared with the host system.
 
 Name | Type | Description | `flatpak override` equivalent
 --- | --- | --- | ---
-Network | Toggle | Allow or prohibit access to the network. | `--share=network` and `--unshare=network`
-[Inter-process communications](https://en.wikipedia.org/wiki/Inter-process_communication) | Toggle | Share or unshare IPC namespace with the host. | `--share=ipc` and `--unshare=ipc`
+Network | Toggle | Allow the program to have access to the network, e.g. if it's disabled for Firefox, you will no longer be able to browse the internet. | `--share=network` and `--unshare=network`
```

Comment:
I personally prefer separating paragraphs. The first one for the description of the option, and the second for the example. Putting them in one paragraph just feels wrong because it's more suitable in essays. What do you think?

Reply:
Sure, let's give a try.

Reply:
https://github.com/tchx84/Flatseal/pull/171/commits/f83ae55dfe911ccb6c523866c5468f42649d13e0